### PR TITLE
MS Dynamics CRM CEQL

### LINKED ISSUE #710: Docathon - Microsoft Dynamics CRM - CEQL
## Description
Update CEQL docs to reflect all known limitations, expectations, and best practices

### PR DESCRIPTION
**Pull request by rrothenb**
## Closes
* Closes #710

### CODE REVIEW

File: docs/elements/dynamicscrm/ceql.md [hunk 1]
```diff
@@ -1,14 +1,107 @@
 ---
 heading: Microsoft Dynamics CRM
-seo: Query Language | MS Dynamics CRM | Cloud Elements API Docs
+seo: Query Language | Microsoft Dynamics CRM | Cloud Elements API Docs
 title: Querying
 description: Make database queries using Cloud Elements Query Language.
 layout: sidebarelementdoc
 breadcrumbs: /docs/elements.html
-elementId: 190
+elementId: 62
```
Comment (davewilks) [inline docs/elements/dynamicscrm/ceql.md]: Holy crap, the element ID was wrong!?

File: docs/elements/dynamicscrm/ceql.md [hunk 2]
```diff
@@ -1,14 +1,107 @@
...
-{% include ceql.md %}
+# Cloud Elements Query Language
+
+The Cloud Elements Query Language (CEQL) is a query language used by Cloud Elements to standardize searching across all of our different elements. Many APIs support some form of searching in their APIs but they’re almost all different, so we have standardized a common way to search across all of our elements. Cloud Elements translates the CEQL to the endpoint’s searching syntax, however at times, CEQL supports more than the endpoint.
+
+## CEQL in {{page.heading}}
+
+When querying in {{page.heading}}, keep the following in mind:
+
+* The {{page.heading}} bulk chunking mechanism allows you to use bulk on large data sets without limitations. If you use an orderBy clause, chunking is disabled. If chunking is enabled, it may take longer to get all the data. Also, be aware that {{page.heading}} has daily upload and download limits.
```
Comment (davewilks) [inline docs/elements/dynamicscrm/ceql.md]: I'll update the template to clarify, but this is more of an example of what could be in here.

File: docs/elements/dynamicscrm/ceql.md [hunk 3]
```diff
@@ -1,14 +1,107 @@
...
+* Use CEQL to query the following endpoints:
```
Comment (davewilks) [inline docs/elements/dynamicscrm/ceql.md]: I didn't check, but is this for sure the list or was it just from the template.

File: docs/elements/dynamicscrm/ceql.md (hunk 1 above)
Comment (rrothenb) [inline docs/elements/dynamicscrm/ceql.md]: Nope.  I screwed up.  It is 190, but only in prod.  It's different in staging.

File: docs/elements/dynamicscrm/ceql.md (hunk 3 above)
Comment (rrothenb) [inline docs/elements/dynamicscrm/ceql.md]: The endpoints were the ones I got from the swagger documentation.  There may be others supported, but they would be covered by /{object}

**Comment by rsbarbo** [comment]
@davewilks just saw that you assigned this PR to me, should I just make sure the changes are proper and approve the PR. Sorry just looking for some guidance on what exactly you are looking for :) thank you - Rapha Barbo

**Pull request merged**